cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -432,8 +439,8 @@
 create_zero_image = True
 """Create zero image, without any parameters"""
 
-images_not_included_in_filter = ['image5']
-"""Images not to be migrated"""
+images_not_to_be_migrated = ['image5', 'image7']
```

Comment:
Instead of extending and complicating logic for images not included in filter, you should rather add another list of images which are blacklisted. This should simplify your code and will remove that crazy functional code in cloudferry_devlab/cloudferry_devlab/tests/utils.py `filter_images`